tools/mo/openvino/tools/mo/convert.py

```diff
@@ -0,0 +1,1205 @@
+    if isinstance(value, LayoutMap):
+        assert value.source_layout is not None, "Incorrect layout map. 'source_layout' should be set."
+        source_layout = layout_to_str(value.source_layout)
+        if value.target_layout is not None:
+            target_layout = layout_to_str(value.target_layout)
+            source_layout += "->" + target_layout
+        return source_layout
+    return layout_to_str(value)
+
+
+def layout_param_to_str(value):
+    # default empty value
+    if isinstance(value, tuple) and len(value) == 0:
+        return value
+
+    if isinstance(value, str):
+        return value
+
+    if isinstance(value, dict):
+        values_str = []
+        for op_name, layout in value.items():
+            if not isinstance(op_name, str):
+                raise Exception("Incorrect operation name type. Expected string, got {}".format(type(op_name)))
+            values_str.append(op_name + "(" + layoutmap_to_str(layout) + ")")
+        return ",".join(values_str)
+
+    raise Exception("Incorrect layout. Expected dictionary, where key is operation name and value is "
+                    "Layout or LayoutMap. Got {}".format(value))
+
+
+def batch_to_int(value):
+    if value is None or isinstance(value, int):
+        return value
+    if isinstance(value, Dimension):
+        if not value.is_static:
+            # TODO: Add dynamic batch support when MO supports dynamic batch
+            raise Exception("Dynamic batch is not supported.")
+        else:
+            return value.get_length()
+    raise Exception("Incorrect batch value. Expected int, got {}.".format(type(value)))
+
+
+def transform_param_value_to_str(value):
+    # This function supports parsing of parameters of MakeStateful, LowLatency2, Pruning.
+    # If available transforms list is extended this method should be extended for new transforms.
+    if isinstance(value, str):
+        return value
+    if isinstance(value, bool):
+        return str(value)
+    if isinstance(value, dict):
+        # param_res_names dictionary for MakeStateful transform
+        values_str = []
+        for input_name, output_name in value.items():
+            assert isinstance(input_name, str), "Incorrect input name. " \
+                                                "Expected string, got {}".format(type(input_name))
+            assert isinstance(output_name, str), "Incorrect output name. " \
+                                                 "Expected string, got {}".format(type(output_name))
+            values_str.append("\'{}\':\'{}\'".format(input_name, output_name))
+        return "{" + ','.join(values_str) + "}"
+    raise Exception("Unknown parameter type.")
+
+
+def transform_to_str(value):
+    if isinstance(value, str):
+        return value
+    if isinstance(value, tuple):
+        assert 1 <= len(value) <= 2, "Incorrect transform. Expected tuple with transform name " \
+                                     "and dictionary with transform parameters. " \
```

Comment:
This comment may confuse user because a tuple is not only a valid type, there are other acceptable types, e.g. `str`. We have to print something like this: f`Incorrect definition of transformation in transform argument: expect two elements in tuple, {len(value)} provided. Supported transform is: ...` and just put a usula doc message. It should be automated at the level of parsing of all parameters instead of tuning this string for each parameter.

It is applicable for all other parameters above and below, no only for this one.

Reply:
Done.